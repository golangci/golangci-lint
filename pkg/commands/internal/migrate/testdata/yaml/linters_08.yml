issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

# case: enable-all + disable + fast
linters:
  fast: true
  enable-all: true
  disable:
    - asasalint
    - asciicheck
    - bidichk
    - bodyclose
    - canonicalheader
    - containedctx
    - contextcheck
    - copyloopvar
    - cyclop
    - decorder
    - depguard
    - dogsled
    - dupl
    - dupword
    - durationcheck
    - err113
    - errcheck
    - errchkjson
    - errname
    - errorlint
    - exhaustive
    - exhaustruct
    - exptostd
    - fatcontext
    - forbidigo
    - forcetypeassert
    - funlen
    - gci
    - ginkgolinter
    - gocheckcompilerdirectives
    - gochecknoglobals
    - gochecknoinits
    - gochecksumtype
    - gocognit

