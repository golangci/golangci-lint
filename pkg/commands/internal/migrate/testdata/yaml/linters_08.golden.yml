version: "2"
linters:
  default: all
  disable:
    - asasalint
    - asciicheck
    - bidichk
    - bodyclose
    - canonicalheader
    - containedctx
    - contextcheck
    - copyloopvar
    - cyclop
    - decorder
    - depguard
    - dogsled
    - dupl
    - dupword
    - durationcheck
    - err113
    - errcheck
    - errchkjson
    - errname
    - errorlint
    - exhaustive
    - exhaustruct
    - exptostd
    - fatcontext
    - forbidigo
    - forcetypeassert
    - funlen
    - ginkgolinter
    - gocheckcompilerdirectives
    - gochecknoglobals
    - gochecknoinits
    - gochecksumtype
    - gocognit
    - gocritic
    - gosec
    - gosmopolitan
    - govet
    - iface
    - importas
    - intrange
    - ireturn
    - loggercheck
    - makezero
    - mirror
    - musttag
    - nilerr
    - nilnesserr
    - nilnil
    - noctx
    - nonamedreturns
    - paralleltest
    - perfsprint
    - protogetter
    - reassign
    - recvcheck
    - revive
    - rowserrcheck
    - sloglint
    - spancheck
    - sqlclosecheck
    - staticcheck
    - tagliatelle
    - testifylint
    - thelper
    - tparallel
    - unconvert
    - unparam
    - unused
    - usetesting
    - varnamelen
    - wastedassign
    - wrapcheck
    - zerologlint
formatters:
  enable:
    - gofmt
    - gofumpt
    - goimports
